packages/cms-cli/lib/commonOpts.js

```diff
@@ -85,10 +85,6 @@ const getAccountId = (options = {}) => {
     return parseInt(process.env.HUBSPOT_PORTAL_ID, 10);
   }
 
-  if (options.useEnv && process.env.HUBSPOT_ACCOUNT_ID) {
```

Comment:
Wasn't sure if we wanted to keep this.  I took it out just for consistency

Reply:
Looks like we should be good to remove this because `process.env.HUBSPOT_PORTAL_ID` is still supported